Sources/CombustionBLE/Utilities/CSV.swift

```diff
@@ -46,6 +46,11 @@ public struct CSV {
         output.append("Framework: iOS")
         output.append("Sample Period: \(probe.temperatureLogs.first?.sessionInformation.samplePeriod ?? 0)")
         output.append("Created: \(dateString)")
+
+        if let sessionDate = probe.temperatureLogs.first?.startTime {
+            output.append("Session date: \(dateFormatter.string(from: sessionDate))")
```

Comment:
Sure - my only concern with this however, is showing an incorrect date. 

Since it's extremely unlikely though, because there must be at least one temperature log, happy to put a fallback to today's date.

Reply:
latest commit address this

Reply:
Instead of an incorrect date, similar to firmware versions,`"??"` would be a good choice.

Reply:
@miwright2 please have a look at the latest commit. I already implemented the fallback with the same approach as the export date, using `??`.

Reply:
done 👍